src/views/Home.vue

```diff
@@ -120,14 +122,15 @@
 import Semester from '../components/Semester.vue';
 import Focus from '../components/Focus.vue';
 import BeautifulProgressIndicator from '../components/BeautifulProgressIndicator.vue';
-import { getColorForCategory } from '../helpers/color-helper';
+import { getColorForCategoryId } from '../helpers/color-helper';
 
-const BASE_URL = 'https://raw.githubusercontent.com/lost-university/data/2.5/data';
+const BASE_URL = 'https://raw.githubusercontent.com/lost-university/data/b177b69a38c4d87fce943b0cfad19580e75e4fdb/data';
```

Comment:
Before merging we need to create a new data release

Reply:
Okey, how would I do that?